src/features/auth/signup/validation.ts

```diff
@@ -0,0 +1,7 @@
+//these are taken from the backend
+export const nameValidationPattern = /\S+/;
+export const usernameValidationPattern = /^[a-z][0-9a-z_]*[a-z0-9]$/;
+
+export function validateBirthdate(stringDate: string) {
```

Comment:
Isn't birthdate too specific?
```suggestion
export function validateDate(stringDate: string) {
```
Or you could also check for a date in the past. But that is not needed, I think.

Reply:
I broke the backend using a future date so added a check (and a bug report)